lib/assessment.rb

```diff
@@ -5,7 +5,13 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.reduce() do |memo, item|
```

Comment:
Correct. Here's a much shorter way you could have written it.
`enum.reduce {|memo, item| memo.length > item.length? memo : item}`